test/scripts/base-host-check.sh

```diff
@@ -80,16 +80,21 @@ get_oscap_score() {
 }
 
 check_ca_cert() {
+    ls /etc/pki/ca-trust/source/anchors
+
+    jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}"
+    jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial
     serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2-)
 
+
     echo "📗 Checking CA cert anchor file"
     if ! [ -e "/etc/pki/ca-trust/source/anchors/${serial}.pem" ]; then
```

Comment:
Looking at https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/8586354060, the `${serial}` is uppercase, while the actual filename is lowercase.